pkg/knuu/knuu_test.go

```diff
@@ -147,3 +161,51 @@ func TestNew(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateOptions(t *testing.T) {
+	tests := []struct {
+		name    string
+		options Options
+		wantErr bool
```

Comment:
this should be `err` and then we set the actual error we expect. This helps enough we are actually hitting the expected error and not just any error.